receiver/k8sobjectsreceiver/receiver.go

```diff
@@ -74,7 +78,7 @@ func newReceiver(params receiver.Settings, config *Config, consumer consumer.Log
 func (kr *k8sobjectsreceiver) Start(ctx context.Context, _ component.Host) error {
 	client, err := kr.config.getDynamicClient()
 	if err != nil {
-		return err
+		return kr.handleError(err, "failed to create dynamic client")
```

Comment:
I guess we missed this one? 
```suggestion
		return err
```

Let's fix it as well.

Reply:
Had it staged, just pushed it, thanks!